frontend/src/components/HomePage/HomePage.tsx

```diff
@@ -32,17 +34,20 @@ function HomePage() {
   return (
     <>
       {!isPanoramaLoaded && <LoadingScreen />}
-      <div id="overlay-root"></div>
+      <div id="overlay-root">
```

Comment:
i may be missing something but we shouldnt need a portal here :) you can just do something like
`{isPanoramaLoaded && <YellowButton text="Play" /> }`